Source/DafnyCore/Options/InMemoryDirectoryInfoFromDotNet8.cs

```diff
@@ -0,0 +1,139 @@
+#nullable enable
```

Comment:
Why the duplicate file (the other copy being `Source/DafnyLanguageServer/Workspace/InMemoryDirectoryInfoFromDotNet8.cs`)? Does it have to do with this `#nullable enable` pragma?

Reply:
Oops !